alea/utils.py

```diff
@@ -117,6 +117,22 @@ def get_analysis_space(analysis_space: list) -> list:
     return eval_analysis_space
 
 
+def _prefix_file_path(config: dict, template_folder_list: list):
+    """Prefix file path with template_folder_list whenever possible.
+
+    Args:
+        config (dict): dictionary contains file path
+        template_folder_list (list): list of possible base folders. Ordered by priority.
+
+    """
+    for key in config.keys():
+        if isinstance(config[key], str):
+            try:
+                config[key] = get_file_path(config[key], template_folder_list)
+            except RuntimeError:
+                pass
```

Comment:
When would you get the RuntimeError?

Reply:
Here https://github.com/XENONnT/alea/blob/4a6f813399eca557ab21a9b198454f2e7c08d2bb/alea/utils.py#L283

Reply:
Ok so basically whenever the str is not a file (or it can't find it but then it will raise an error later)? Makes sense then to silence this.

Reply:
Yes, I agree.